tensorflow_cloud/deploy.py

```diff
@@ -154,6 +156,33 @@ def _print_logs_info(job_id, project_id):
           .format(job_id, project_id))
 
 
+def _stream_logs(job_id):
+    """Streams job logs to stdout.
+
+    Args:
+        job_id: The job id to stream logs from.
+
+    Raises:
+        RuntimeError, if there are any errors from the streaming subprocess.
+    """
+    try:
+        print('Streaming job logs: ')
+        process = subprocess.Popen(
```

Comment:
"The recommended approach to invoking subprocesses is to use the run() function for all use cases it can handle. For more advanced use cases, the underlying Popen interface can be used directly."

Should you use `run` instead? I don't really have appropriate info to judge here.

Reply:
run is available only in Python >= 3.5 hence i used Popen. Afaik the code so far works in python >= 2.7. I am not sure if we want to add version restriction for this since run is only a simplified version of Popen.

Reply:
Python 2.7 is past end of life. We do not intend to support it. If you can use `run`, then please do.

Reply:
Updated.